partcad/src/partcad/project_factory_git.py

```
@@ -220,7 +218,9 @@ def _clone_or_update_repo(self, repo_url, cache_dir=None):
                     # Clone the repository if it's not cached yet.
                     try:
                         pc_logging.info("Cloning the GIT repo: %s" % self.import_config_url)
-                        repo = Repo.clone_from(repo_url, cache_path, multi_options=self.git_config_options, allow_unsafe_options=True)
+                        repo = Repo.clone_from(
+                            repo_url, cache_path, multi_options=self.git_config_options, allow_unsafe_options=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)